backend/endpoints/threads.py

```diff
@@ -5,9 +5,10 @@
 from schema.thread_response import ThreadResponse
 from fastapi.responses import JSONResponse
 from utils.centrifugo import Events, centrifugo_client
-from utils.message_utils import get_message, get_room_messages, update_message_threads as edit_message_threads
+from utils.message_utils import get_message, get_room_messages
 from utils.room_utils import get_org_rooms
 from utils.threads_utils import get_message_threads, add_message_to_thread_list
+from utils.threads_utils import  update_thread_message as edit_thread_message
```

Comment:
why is the alias neccessary?

Reply:
@Shopiley, please rename the endpoint function to **update_thread_message** and that would make sense.